stress/shell/src/main/java/alluxio/stress/cli/MasterBatchTask.java

```diff
@@ -98,7 +115,7 @@ private String getDescription() {
             + " alluxio:///stress-master-base",
         "$ bin/alluxio runClass alluxio.stress.cli.BatchTaskRunner"
             + " MasterComprehensiveFileBatchTask --num-files 1000 --threads 10 --create-file-size"
-            + " 1k --base alluxio:///stress-master-base",
+            + " 1k --base alluxio:///stress-master-base --warmup 0s",
```

Comment:
Is `--warmup 0s` the preferred configuration? If so, you may want to print a warning or something if you see `mParameter.mWarmup` is not 0, and explain in the warning message.

Reply:
I added a warning. Since the file operated in warmup time will not be calculated in the final result, using warmup will make the number of operations less than expected